app/Http/Controllers/SettingsController.php

```diff
@@ -907,7 +907,7 @@ public function postLdapSettings(Request $request)
     public function getBackups()
     {
 
-        $path = storage_path().'/app/'.config('laravel-backup.backup.name');
+        $path = storage_path().'/app/'.config('backup.backup.name');
```

Comment:
This change sticks out to me, are we sure this is what's intended?

Reply:
Yep - they changed the name of their config file.